resources/views/users/admin/overview.blade.php

```diff
@@ -9,19 +9,36 @@
     <div class="row justify-content-center">
 
         <div class="col-md-3">
+
             <form method="get" action="{{ route('user::admin::list') }}">
-                <div class="card">
+                <div class="card mb-4">
                     <div class="card-header bg-dark text-white">Search in users</div>
                     <div class="card-body">
-                        <div class="input-group">
-                            <input type="text" class="form-control" placeholder="Search term" name="query"
-                                   value="{{ $query }}">
-                            <div class="input-group-append">
-                                <button type="submit" class="input-group-text">
-                                    <i class="fas fa-search"></i>
-                                </button>
+                        <div class="input-group mb-2">
+                            <input type="text" class="form-control" placeholder="Search term" name="query" value="{{ $query }}">
+                        </div>
+                        <b>Filter users</b>
+                        <div class="input-group mb-2">
+                            <div class="form-check mr-2">
+                                <input class="form-check-input" type="radio" name="filter" id="users" value="users" @if($filter == 'users') checked @endif>
+                                <label class="form-check-label" for="users"> Users </label>
+                            </div>
+                            <div class="form-check mr-2">
+                                <input class="form-check-input" type="radio" name="filter" id="members" value="members" @if($filter == 'members') checked @endif>
+                                <label class="form-check-label" for="members"> Members </label>
+                            </div>
+                            <div class="form-check mr-2">
+                                <input class="form-check-input" type="radio" name="filter" id="pending" value="pending" @if($filter == 'pending') checked @endif>
+                                <label class="form-check-label" for="pending"> Pending </label>
+                            </div>
+                            <div class="form-check">
+                                <input class="form-check-input" type="radio" id="all" name="filter" value="" @if($filter == null) checked @endif>
+                                <label class="form-check-label" for="all"> All </label>
```

Comment:
Checkboxes hint to the possibility of multiple filters (for example all members and pending members but not normal users) but this doesn't seem possible with the current implementation.

Reply:
That is why I used radio buttons

Reply:
nvm then